YoCode/FileImport.cs

```diff
@@ -27,10 +27,20 @@ public static IEnumerable<string> GetAllFilesInDirectory(string path)
             }
         }
 
-        //Helper method to convert FileInfo[] elements to string and add them to a list 
-        public static void AddFileInfoToList(List<string> files, IEnumerable<FileInfo> fileinfo)
+        public static void AddFileInfoToList(List<string> files, IEnumerable<FileInfo> fileinfo, string path)
         {
-            files.AddRange(fileinfo.Select(fi => fi.ToString()));
+            foreach (var f in fileinfo)
+            {
+                if (FileIsNotInBuildDirectory(path, f))
+                {
+                    files.Add(f.ToString());
+                }
+            }
+        }
+
+        private static bool FileIsNotInBuildDirectory(string path, FileInfo fileinfo)
+        {
```

Comment:
I feel like we now have 2 systems for ignoring files. This has "obj" and "bin" listed as explicit paths to ignore but the code in PathManager's constructor uses `.gitignore` to do basically the same thing. I don't feel like trying to fix that right now but it's something we could probably tidy up at some point.